Plugins/BuildServerIntegration/AzureDevOpsIntegration/BuildsCache.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Collections.Generic;
+using GitUIPluginInterfaces.BuildServerIntegration;
+
+namespace AzureDevOpsIntegration
+{
+    public class BuildsCache
```

Comment:
Are any of the fields get mutated after the initial initialisation? I couldn't quite see it in the browser.
If not, should we change this into a [record](https://docs.microsoft.com/dotnet/csharp/language-reference/builtin-types/record)?

Reply:
Yes, at least one is mutated so not possible.